core/src/main/java/org/n52/iceland/ogc/filter/AbstractProjectionClause.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015 52°North Initiative for Geospatial Open Source
+ * Software GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.n52.iceland.ogc.filter;
+
+/**
+ * interface for FES 2.0 AbstractProjectionClause
+ *
+ * @author <a href="mailto:c.hollmann@52north.org">Carsten Hollmann</a>
+ *
+ * @since 1.0.0
+ *
+ */
+public interface AbstractProjectionClause {
```

Comment:
That's the name of the element in the spec... so I suppose keeping the name is more important than following Java practices. @CarstenHollmann ?